doc/os-image-poll-and-update.md

```diff
@@ -84,4 +84,17 @@ Or on CRC:
 * oc import-image cirros-is -n openshift-virtualization-os-images --from=kubevirt/cirros-container-disk-demo --scheduled --confirm
 * oc set image-lookup cirros-is -n openshift-virtualization-os-images
 
-More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
\ No newline at end of file
+More information on image streams is available [here](https://docs.openshift.com/container-platform/4.8/openshift_images/image-streams-manage.html) and [here](https://www.tutorialworks.com/openshift-imagestreams).
```

Comment:
Suggest updating this link: https://docs.openshift.com/container-platform/4.13/openshift_images/image-streams-manage.html

Reply:
done, thx